### PR TITLE
Blog app - Views

### PR DESCRIPTION
**Pull request by amrendrakind**
### Views: Project requirements

- Implement the design from the sneak peek wireframes.
    - Styling is nice to have but focus on making sure everything functions first.
- Use methods that you have created in the [Blog App - processing data in models](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md).
- **Hint**: In order to see some posts or comments, you might need to add some data to your database as described [here](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md#use-models-to-insert-data).

### CODE REVIEW

File: app/controllers/logins_controller.rb [hunk 1]
```diff
@@ -0,0 +1,69 @@
+class LoginsController < ApplicationController
+  before_action :set_login, only: %i[show edit update destroy]
+
+  # GET /logins or /logins.json
+  def index
+    @logins = Login.all
+  end
+
+  # GET /logins/1 or /logins/1.json
```
Comment (Davidosky007) [inline app/controllers/logins_controller.rb]: * Good job implementing some of the views according to the wireframes provided, but in the [project requirement ](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Implement%20the%20design,everything%20functions%20first)you are to implement also the login page, Kindly fix this 🙏 below is a screenshot of the view ⬇️ 
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @amrendrakind 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
* No linter errors 👍 
* Well written readme file 👍 
* Good looking PR description 👍 
* Nice job using partials for your views 👏 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/logins_controller.rb (hunk 1 above)
Comment (amrendrakind) [reply app/controllers/logins_controller.rb]: @Davidosky007 Thank tou for your valuable feedback. I will implement recommendations.

**Review by rustam-tolipov — APPROVED** [review]
## Hello @amrendrakind 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**